### DISCUSSION TITLE
Ask for help :  Building Scada-LTS version 2.7.1.1 on windows10

### DISCUSSION THREAD

**Discussion by Ch0pperX** [SCADA-LTS/Scada-LTS#2610] [Q&A]
Hi, 
I would like to build scada-LTS version 2.7.1.1 on window 10. I tried to follow the tips from the official site and used the gradle(version 8.2.1) to build it. It still had some error that I couldn't debug by myself. Would anyone help me to build the application?

**Comment by Limraj** [comment]
Hi @Ch0pperX,

We're in the process of creating a manual for windows, I'm working on it @Patrykb0802 .

However, firstly, you are building branch develop:
```
git checkout develop
git pull
```
Secondly, the gradle version should be max 7.3, [node 14.20.1](https://nodejs.org/download/release/v14.20.1/). Here you have a list of versions in the table, the key is the version: java, gradle, node, tomcat, mysql, the rest: git, intelliej can be the latest:
[Getting-started-Development](https://github.com/SCADA-LTS/Scada-LTS/wiki/Getting-started-Development)

Simple instruction:
1. To install the runtime/client (without node, gradle, intellij), you can use the [installer](https://github.com/SCADA-LTS/windows-installer).
2. Next, set the environment variables JAVA_HOME (unless it is) and CATALINA_HOME absolute path to the folder "(...)Program Files/Scada-LTS/tomcat" There is a tomcat from the installer that you can use. The installer also has a mysql database available on port 3308.
3. Next, uninstall too new node versions and install the correct one, and use the correct version of gradle, and we support jave not newer than 11. 

Regards,
Kamil Jarmusik
